healthlocker_service_script.md

```diff
@@ -0,0 +1,107 @@
+The first step is to open your terminal and make sure that you are in the root
+of the healthlocker Repo. Once here run the following...
+
+```
+cat healthlocker.service
+```
+
+This will print the contents of the file into your terminal. Highlight the
+contents and copy them to your clipboard with `cmd/control + c`
+
+The next step is to `ssh` onto the server with
+```
+ssh root@'ip_address'
+```
+
+Once on the server create a file called `'app_name'.service` in the
+`/lib/systemd/system/` directory. For example
+
+```
+touch /lib/systemd/system/healthlocker.service
+```
+
+Now that the file is created you will need to paste the code you have copied
+to your clipboard into the file. To do this you will need to...
+
+- vim /lib/systemd/system/'app_name'.service
+- type :set paste and then press return
+- type i to enter insert mode
+- paste the contents you copied earlier
+- press the escape key
+
+You will need to update the `Description` and `WorkingDirectory` lines with the
+necessary information. To do so
+
+- use the arrow keys to go to the end of the line you want to edit
+- press i to ender insert mode
```

Comment:
Typo!

Reply:
😭 Thanks for spotting that @Cleop

Reply:
And that is a good point thanks. Will update now